### PR TITLE
User Dashboard GitHub

### PR DESCRIPTION
**Pull request by danbriechle**
Hides tokens 
Adds edge case tests 
Adds link to user show

### CODE REVIEW

File: spec/features/user/logged_in_user_sees_git_spec.rb [hunk 1]
```diff
@@ -14,20 +14,50 @@
...
+
+        within ".top5repos" do
+          expect(page).to have_content("little-shop")
+          expect(page).to have_link("little-shop")
+        end
+      end
+    end
+
+    it 'only sees its git hub info ' do
+      VCR.use_cassette("nico_cassette") do
```
Comment (nicovigil1) [inline spec/features/user/logged_in_user_sees_git_spec.rb]: clever!
